modules/channel-web/src/views/lite/store/index.ts

```diff
@@ -428,14 +429,28 @@ class RootStore {
 
     document.title = this.config.botName || 'Botpress Webchat'
 
+    this.api.updateAxiosConfig({ botId: this.config.botId, externalAuthToken: this.config.externalAuthToken })
+    this.api.updateUserId(this.config.userId)
```

Comment:
are those 2 promises ? do we want to await them ?

Reply:
They aren't and were simply moved from the bottom of this function to before the if statement.